core/object/script_language_extension.h

```diff
@@ -474,6 +474,13 @@ class ScriptLanguageExtension : public ScriptLanguage {
 		return result;
 	}
 
+	GDVIRTUAL1RC(String, _format_code, const String &)
+	virtual String format_code(const String &p_code) override {
+		String ret;
+		GDVIRTUAL_CALL(_format_code, p_code, ret);
+		return ret;
```

Comment:
Wouldn't that format everything to an empty string if a language does not implement format code?